---
layout: page
title: ""
---

### Backgound 
I am an immunologist with ~10 years of experience in cellular immunology, computational biology, and transcriptomic data analysis.
Utilizing my hybrid research profile with both bench (wet lab) and bioinformatics (dry lab) expertise, I have focused my efforts towards understanding immune system dynamics in different contexts including cancer, pregnancy, aging, and autoimmune diseases. Many of these studies were performed in collaboration with other scientists. 

### Ph.D. work: 
During my doctoral (Klatzmann lab; University Pierre & Marie Curie, Paris, France; 2010-2016), my research focused on understanding why embryos and tumors expressing foreign antigens were not rejected by the immune system. Using murine models (pregnant mice and mice injected with different tumor models), I demonstrated that a specific CD4+ T cell population, the regulatory T cells (Tregs), orchestrates similar immune evasion mechanisms in both fetuses and tumors. Next, I investigated how key genes can regulate the immune response to tumors by generating tumor models of TGF-β and/or VEGF silencing. Using both transcriptomic profiling and flow cytometry analysis of the tumor microenvironment, I demonstrated that co-silencing of TGF-β and VEGF led to a substantial spontaneous tumor eradication rate and the combination of their respective inhibitory drugs was synergistic. Additionally, I established a collaboration with the Aryee lab (Massachusetts General Hospital & Harvard Medical School) to confirm my findings on the similarities between tumor and fetal development in humans. Analyzing both transcriptomic and DNA methylation profiles, we were able to demonstrate that the human placenta during early pregnancy shares the patterns of DNA hypomethylation as seen in tumors.

### Postdoctoral work:
Within the Banchereau lab (2016-present), I have been developing computational pipelines to analyze human blood cell transcriptomic data at the single cell resolution in different contexts including systemic lupus erythematosus (SLE), aging, and influenza vaccination. My initial work sought to resolve the transcriptional signatures described decades ago in SLE using cutting-edge single cell RNA-seq approaches. This analysis allowed a more accurate classification of lupus patients based on specific cell types. This first direct comparison of childhood and adult SLE confirmed the presence of similar transcriptional patterns across age groups and enabled patient classification according to disease activity.

Currently, I am a Reseach Scientist at The Jackon laboratory for Genomic Medecine (Verhaak lab).
