---
title:  "Mapping systemic lupus erythematosus heterogeneity at the single-cell level"
mathjax: true
layout: post
categories:
  - github
  - website
---


Nehar-Belaid et al. Nature Immunology, 2020. PMID: 32747814

## Abstract 
Patients with systemic lupus erythematosus (SLE) display a complex blood transcriptome whose cellular origin is poorly
resolved. Using single-cell RNA sequencing, we profiled ~276,000 peripheral blood mononuclear cells from 33 children with
SLE with different degrees of disease activity and 11 matched controls. Increased expression of interferon-stimulated genes
(ISGs) distinguished cells from children with SLE from healthy control cells. The high ISG expression signature (ISGhi) derived
from a small number of transcriptionally defined subpopulations within major cell types, including monocytes, CD4+ and CD8+
T cells, natural killer cells, conventional and plasmacytoid dendritic cells, B cells and especially plasma cells. Expansion of
unique subpopulations enriched in ISGs and/or in monogenic lupus-associated genes classified patients with the highest disease
activity. Profiling of ~82,000 single peripheral blood mononuclear cells from adults with SLE confirmed the expansion of
similar subpopulations in patients with the highest disease activity. This study lays the groundwork for resolving the origin of
the SLE transcriptional signatures and the disease heterogeneity towards precision medicine applications. 

[Link here]


## Data avaibility 

#### 1- Raw counts (publicly available): 
GEO link: [GSE135779]


#### 2- fastq files (to be requested on dbGAP): 
dbGAP link: [phs002048.v2.p1]

## Scripts availability 
[github.com/dnehar/SingleCells_SLE_paper]


## Context 
In an effort towards understanding the SLE heterogeneity, we used single-cell RNA-seq to profile ~276,000 PBMCs from 33 children with SLE with different degrees of disease activity (DA) and 11 matched controls. We demonstrated that the interferon-stimulated genes signature derived from a small number of transcriptionally defined subpopulations within major cell types including monocytes, CD4+ and CD8+ T cells, naturel killer cells, conventional and plasmacytoid dendritic cells, B cells and especially plasma cells. Expansion of unique subpopulations enriched in ISGs and/or in monogenic Lupus-associated genes classified patients with the highest DA. In addition, the profiling of ~82,000 single PBMCs from an independent adult SLE patients confirmed the expansion of similar subpopulations in patients with the highest DA. This study provided a framework for resolving the origin of the SLE transcriptional signature and point towards specific cell subpopulations as potential therapeutic targets.


## Design of childhood-adult cohort (caSLE)

![ExperimetalDesign](https://user-images.githubusercontent.com/22060632/147284489-52add15a-3997-4690-89e6-bc0c148c36cb.png)


[GSE135779]: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135779
[Link here]: https://doi.org/10.1038/s41590-020-0743-0
[phs002048.v2.p1]: https://www.ncbi.nlm.nih.gov/projects/gapprev/gap/cgi-bin/study.cgi?study_id=phs002048.v2.p1

[github.com/dnehar/SingleCells_SLE_paper]: https://github.com/dnehar/SingleCells_SLE_paper
